lib/helpers.js

```diff
@@ -90,6 +91,28 @@ function getIMEListFromOutput (stdout) {
   return engines;
 }
 
+async function getSdkToolsVersion() {
```

Comment:
wouldn't it be easier to just read the whole file and parse it with regexp (or even simply execute _grep_ from commend line)? 

Also, there are node modules for property files parsing, like https://www.npmjs.com/package/properties-parser We could include them as a dependency.